src/Backend/Modules/Groups/Engine/Model.php

```diff
@@ -53,6 +53,9 @@ public static function alreadyExists(string $groupName): bool
     public static function delete(int $groupId): void
     {
         BackendModel::getContainer()->get('database')->delete('groups', 'id = ?', [$groupId]);
+        BackendModel::getContainer()->get('database')->delete('groups_settings', 'group_id = ?', [$groupId]);
```

Comment:
Please add `BackendModel::getContainer()->get('database')` to a variable so you don't have to go through all those getters every time.

```
/** @var SpoonDatabase $database */
$database = BackendModel::getContainer()->get('database');
```